<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qthuffman</class>
 <widget class="QMainWindow" name="Qthuffman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qthuffman</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Encode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select the txt file you want to compress</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select the txt file you want to decompress</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select the save location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enocdeSrcBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>source</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decodeSrcBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>source</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decodeDstBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>destination</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encodeBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>encode</string>
    </property>
   </widget>
   <widget class="QLabel" name="encodeSrcLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="decodeSrcLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="decodeDstLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="encodeDstBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>destination</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decodeBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>decode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select the save location</string>
    </property>
   </widget>
   <widget class="QLabel" name="encodeDstLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHuffman">
    <property name="title">
     <string>Huffman</string>
    </property>
   </widget>
   <addaction name="menuHuffman"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
